_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -78,92 +77,101 @@ Lets take a look at the pinger HTML, and see how it works.
     <title>Pinger</title>
     <meta charset="utf-8">
     <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
-    <script src="../base1/jquery.js"></script>
     <script src="../base1/cockpit.js"></script>
 </head>
 <body>
-    <div class="container-fluid" style='max-width: 400px'>
-        <table class="cockpit-form-table">
+    <div class="container-fluid">
+        <table class="form-table-ct">
             <tr>
-                <td>Address</td>
+                <td><label class="control-label" for="address">Address</label></td>
                 <td><input class="form-control" id="address" value="8.8.8.8"></td>
             </tr>
             <tr>
-                <td><button class="btn btn-primary" id="ping">Ping</button></td>
+                <td><button class="pf-c-button pf-m-primary" id="ping">Ping</button></td>
                 <td><span id="result"></span></td>
             </tr>
         </table>
-        <p><pre id="output"></pre></p>
+        <pre id="output"></pre>
     </div>
-    <script>
-        var address = $("#address");
-        var output = $("#output");
-        var result = $("#result");
 
-        $("#ping").on("click", ping_run);
+    <script src="pinger.js"></script>
+</body>
+</html>
 
-        function ping_run() {
-            var proc = cockpit.spawn(["ping", "-c", "4", address.val()]);
-            proc.done(ping_success);
-            proc.stream(ping_output);
-            proc.fail(ping_fail);
+```
 
-            result.empty();
-            output.empty();
-        }
+The page includes the cockpit.css file to make sure the look of our tool matches that of Cockpit.
 
-        function ping_success() {
-            result.css("color", "green");
-            result.text("success");
-        }
+Then we include `cockpit.js`: the basic API for interacting with the system, as well as Cockpit itself. You can find [detailed documentation here](https://cockpit-project.org/guide/latest/api-cockpit.html).
```

Comment:
```suggestion
Here we include `cockpit.js`: the basic API for interacting with the system, as well as Cockpit itself. You can find [detailed documentation in the Cockpit guide](https://cockpit-project.org/guide/latest/api-base1.html).
```

(Changed the link to point to the contents above the page, as the `api-cockpit` page is mostly empty and the `api-base1` page is mainly documentation on `cockpit.js`.)